app/cli/internal/telemetry/telemetry_test.go

```diff
@@ -0,0 +1,145 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package telemetry_test
+
+import (
+	"context"
+	"runtime"
+	"testing"
+
+	"github.com/chainloop-dev/chainloop/app/cli/internal/telemetry"
+	"github.com/chainloop-dev/chainloop/app/cli/internal/telemetry/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestTags_WithEnvironmentInfo(t *testing.T) {
```

Comment:
is using underscores `idiomatic` in Go? I am not used to see it.

Reply:
I've seen using it in subtests so if you have `TestNewClient` and another other related to the first, the second goes in underscore:
`TestNewClient` -> `TestNewClient_WithTags`